### DISCUSSION TITLE
The feasibility of port F# computation expressions to v

### DISCUSSION THREAD

**Discussion by Lycs-D** [vlang/v#19638] [Ideas]
[Computation Expressions](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/computation-expressions)
[Creating a New Type of Computation Expression](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/computation-expressions#creating-a-new-type-of-computation-expression)

**Comment by alfu32** [comment]
what is the difference to lambdas or function references ?
